### ISSUE TITLE
Alphabetize Seeds List

### ISSUE THREAD

**Issue by bkonia** [HermesApp/Hermes#333]
Would be nice if the seeds list could be alphabetized.

**Comment by nriley** [comment]
Agreed. It is a bit harder than with the likes/dislikes as this is an outline as opposed to just a list, but would not be difficult.
